tests/ci.rs

```diff
@@ -1,52 +1,10 @@
-use gh_workflow::toolchain::Toolchain;
-use gh_workflow::*;
+use gh_workflow_tailcall::*;
 
 #[test]
-fn generate() {
-    let flags = RustFlags::deny("warnings");
+fn generate_ci_workflow() {
+    let workflow = Workflow::default()
+        .auto_release(true)
+        .name("Build and Test".into());
```

Comment:
```suggestion
        .benchmarks(true);
```

Something like this would be better.